driver/src/models/standing_order.rs

```diff
@@ -19,19 +19,17 @@ impl StandingOrder {
     pub fn get_orders(&self) -> &Vec<super::Order> {
         &self.orders
     }
+    pub fn get_orders_length(&self) -> usize {
```

Comment:
could we call this `num_order`? Then using it would be `standing_order.num_orders()`

Reply:
yeah, that is better...